### PR TITLE
Updated the guide for setting up the MongoDB database

### LINKED ISSUE #15556: Outdated guide for setting up MongoDB database
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/mongoose

### What specific section or headline is this issue about?

Setting up the MongoDB database

### What information was incorrect, unhelpful, or incomplete?

The user interface of MongoDB has changed. The screenshots and the guide on the page still refer to the old interface which could lead to confusion among learners. 

Correcting this would require modifying the instruction and changing all screenshots under the [Setting up the MongoDB database](https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/mongoose#setting_up_the_mongodb_database) section. Here is a side-by-side comparison of one of the steps, the left image is the one in the guide, right image is how the UI currently looks on MongoDB:

comparison

I would like to work on this and make a PR if that's okay.

### What did you expect to see?

Updated information about how to set up the MongoDB database.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/server-side/express_nodejs/mongoose`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Express_Nodejs/mongoose
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/server-side/express_nodejs/mongoose/index.md
* Last commit: https://github.com/mdn/content/commit/831aabc34db752bfd7fac701577870d7692b2804
* Document last modified: 2022-04-28T05:11:47.000Z

</details>

### PR DESCRIPTION
**Pull request by alabhyajindal**
#### Summary

Updated the guide to match the current UI of MongoDB. Replaced all screenshots with new ones. Updated the steps to match the new screenshots. I have used the earlier content wherever I could and wrote new where needed.

#### Motivation

To help readers easily set up the MongoDB database with the updated instructions.

#### Related issues

Fixes [15556](https://github.com/mdn/content/issues/15556)

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by hamishwillee** [comment]
@alabhyajindal Thanks very much!!!! 

You can see that some of the checks failed. That's because our system requires that images be compressed using the inbuilt tool. For example, in the log it suggests you run this command:
```
yarn filecheck "files/en-us/learn/server-side/express_nodejs/mongoose/mongodb_atlas_-_accessrules.jpg" --save-compression
```
If you could do that for your new images that would be great. I'll look at the content now.

**Comment by hamishwillee** [comment]
For the image of the tier you select, could you please take the screenshot with the shared database selected, like this:
image

That way the image matches what you are asking the user to do.

File: files/en-us/learn/server-side/express_nodejs/mongoose/index.md [hunk 1]
```diff
@@ -518,68 +518,69 @@ You will first need to [create an account](https://www.mongodb.com/cloud/atlas/r
...
+    
+2. This will open the _Deploy a cloud database_ screen. Click on the **Create** button under the _Shared_ deployment option.
+    ![Choose a deployment option when using MongoDB Atlas.](mongodb_atlas_-_deploy.jpg)
 
-    - Select any provider from the *Cloud Provider & Region* section. Different providers offer different regions.
-    - Select any region marked "FREE TIER AVAILABLE".
+3. This will open the _Create a Shared Cluster_ screen.
+    ![Choose a cloud provider when using MongoDB Atlas.](mongodb_atlas_-_createsharedcluster.jpg)
+
+    - Select any provider from the _Cloud Provider & Region_ section. Different providers offer different regions.
```
Comment (hamishwillee) [inline files/en-us/learn/server-side/express_nodejs/mongoose/index.md:530]:
```suggestion
    - Select any provider from the _Cloud Provider & Region_ section. Different regions offer different providers.
```

File: files/en-us/learn/server-side/express_nodejs/mongoose/index.md [hunk 2]
```diff
@@ -518,68 +518,69 @@ You will first need to [create an account](https://www.mongodb.com/cloud/atlas/r
...
-    ![Configure a connection when after setting up a cluster in MongoDB Atlas.](mongodb_atlas_-_connectbutton.jpg)
-8. This will open the Connect to Cluster screen.
-    ![Choose a connection type when connecting with MongoDB Atlas.](connect_to_cluster_allowanywhere.png)
+7. This will open the _Collections_ section. Click the **Add My Own Data** button.
+    ![Create a database on MongoDB Atlas.](mongodb_atlas_-_adddata.jpg)
 
-    - Click the **Allow Access from Anywhere** button.
-      This will open a form with `0.0.0.0/0` pre-seeded for the IP Address. Click the **Add IP Address** button.
+8. This will open the _Create Database_ screen.
+    ![Details during database creation on MongoDB Atlas.](mongodb_atlas_-_databasedetails.jpg)
```
Comment (hamishwillee) [inline files/en-us/learn/server-side/express_nodejs/mongoose/index.md:557]:
```suggestion

   ![Details during database creation on MongoDB Atlas.](mongodb_atlas_-_databasedetails.jpg)
```

**Comment by hamishwillee** [comment]
Looks great otherwise. I'm back Friday and will look at your updates then! Thanks again.

**Review by hamishwillee — APPROVED** [review]
@alabhyajindal - excellent. Thank you - this is a really helpful update in my opinion.

**Comment by alabhyajindal** [comment]
@hamishwillee Thanks a lot!! Your suggestions earlier when the checks were failing were very helpful 😁

**Pull request merged**